docs/LogGuide.md

```diff
@@ -63,7 +63,7 @@ This guide was last updated for:
     - [Structure](#structure-7)
     - [Regexes](#regexes-7)
     - [Examples](#examples-7)
-    - [Cast Times](#cast-times)
+  - [Cast Times](#cast-times)
```

Comment:
```suggestion
    - [Cast Times](#cast-times)
```
The indentation is important for sub-list ordering

Reply:
Fixed